packages/widget-chat/package.json

```diff
@@ -51,15 +50,13 @@
     "jest": "^15.1.1",
     "postcss": "^5.2.0",
     "postcss-cssnext": "^2.8.0",
-    "postcss-loader": "^0.13.0",
     "postcss-reporter": "^1.4.1",
     "react-addons-test-utils": "^15.3.1",
     "react-test-renderer": "^15.3.1",
     "redux-logger": "^2.6.1",
     "redux-mock-store": "^1.2.1",
     "redux-thunk": "^2.1.0",
-    "require-dir": "^0.3.0",
-    "style-loader": "^0.13.1"
+    "require-dir": "^0.3.0"
```

Comment:
No idea. I just removed the comma from the end of the line. Think it was a leftover package.